packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -95,7 +95,11 @@ export interface SdkClientType<TServiceOperation extends SdkServiceOperation>
   clientNamespace: string; // fully qualified namespace
   doc?: string;
   summary?: string;
+  /**
+   * @deprecated Use `clientInitialization` instead.
+   */
   initialization: SdkInitializationType;
+  clientInitialization: SdkClientInitializationMethod;
```

Comment:
It is a bit strange that `SdkClientInitializationMethod` is here, where `SdkClientAccessor<>` appears in `methods`?

Reply:
i also want to move client assessor method out of the method list. just tbd.